### PR TITLE
Add `Sign Bulk Transaction` button

### PR DESCRIPTION
**Pull request by dhriaznov**
This PR adds a `Sign Bulk Transaction` button to test the web extension functionality made in the scope of https://linear.app/xverseapp/issue/ENG-2879/batch-sign-psbt-in-extension

### CODE REVIEW

**Review by m-aboelenein — APPROVED** [review]

**Review by DuskaT021 — COMMENTED** [review]
@dhriaznov it will compile with errors if you leave the `^` next to the sats-connect version. I'm testing without the `^`

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
-    "sats-connect": "^1.1.1-bd7b944",
+    "sats-connect": "^1.1.2-5f6a296",
```
Comment (DuskaT021) [inline package.json]: compiles with errors if there is a `^` next to sats connect version. I'm testing with out the carrot

**Comment by teebszet** [comment]
@dhriaznov please check if the sats-connect version is correct, and if this is ready for merge

**Comment by dhriaznov** [comment]
@teebszet should we merge the sats-connect first and then update its' version in this PR?

**Pull request merged**